x-pack/plugins/apm/typings/common.d.ts

```diff
@@ -5,11 +5,7 @@
  * 2.0.
  */
 
-import '../../../typings/rison_node';
```

Comment:
I think this is a file we use for our optimized tsconfig setup. But we can fix it later.